envs/vpn/conda/fedbiomed-node.yaml

```diff
@@ -10,56 +10,55 @@ channels:
 
 dependencies:
   # minimal environment
-  - python >=3.9,<3.10
-  - pip
-  - jupyter
-  - ipython
+  # python 3.11 recently released 2022-11 and not yet supported by some deps including: torchvision
+  - python >=3.10,<3.11
+  - pip >= 23.0
+  - jupyter ~=1.0.0
+  - ipython ~=8.13.2
```

Comment:
Some comment for `ipython` i don't whether it is really necessary since it is never used on the node side.

Reply:
OK, also removing `ipython` for the 3 node environments

Reply:
We introduced a regression with that one, see #803 (will be corrected through this PR)